tests/test_create.py

```diff
@@ -191,7 +191,7 @@ def test_remove_all(self):
             run_command(Commands.REMOVE, prefix, '--all')
             assert not exists(prefix)
 
-    @pytest.mark.skipif(on_win, reason="no 32-bit windows python on conda-forge")  # and bits == 32
+    @pytest.mark.skipif(on_win and bits == 32, reason="no 32-bit windows python on conda-forge")
```

Comment:
I thought we did have Windows 32-bit. 😕

Could you please share some more context as to what motivated this?